### PR TITLE
Issue #8328: Change name of checkstyle types to be close to real java types - other types

### PR DESCRIPTION
**Pull request by gaurabdg**
#8328 

Types added to `property_types.xml`:

-  byte, byte[]
- char, char[]
- float, float[]
- long, long[]
- short, short[]

### CODE REVIEW

File: src/xdocs/property_types.xml [hunk 1]
```diff
@@ -30,6 +30,101 @@
...
+        This type represents a char. The string representation is
+        parsed using the <code>java.lang.Character</code> class.
+      </p>
+    </section>
+
+    <section name="char[]">
+      <p>
+        This type represents a set of chars. The string representation
+        is parsed as a set of comma (',') separated chars that are parsed
+        using the <code>java.lang.Char</code> class.
```
Comment (strkkk) [inline src/xdocs/property_types.xml:63]: should be Character

File: src/xdocs/property_types.xml [hunk 2]
```diff
@@ -30,6 +30,101 @@
...
+      </p>
+      <p>
+        Alternatively, a property of this type can be supplied multiple times which
+        is equivalent to a set of comma separated byte values.
```
Comment (strkkk) [inline src/xdocs/property_types.xml:67]: `byte` -> char

File: src/xdocs/property_types.xml [hunk 3]
```diff
@@ -30,6 +30,101 @@
...
+
+    <section name="float[]">
+      <p>
+        This type represents a set of floats. The string representation
+        is parsed as a set of comma (',') separated floats that are parsed
+        using the <code>java.lang.Float</code> class.
+      </p>
+      <p>
+        Alternatively, a property of this type can be supplied multiple times which
+        is equivalent to a set of comma separated byte values.
```
Comment (strkkk) [inline src/xdocs/property_types.xml:86]: same here

File: src/xdocs/property_types.xml [hunk 4]
```diff
@@ -30,6 +30,101 @@
...
+
+    <section name="long[]">
+      <p>
+        This type represents a set of longs. The string representation
+        is parsed as a set of comma (',') separated longs that are parsed
+        using the <code>java.lang.Long</code> class.
+      </p>
+      <p>
+        Alternatively, a property of this type can be supplied multiple times which
+        is equivalent to a set of comma separated byte values.
```
Comment (strkkk) [inline src/xdocs/property_types.xml:105]: same here

File: src/xdocs/property_types.xml [hunk 5]
```diff
@@ -30,6 +30,101 @@
...
+
+    <section name="short[]">
+      <p>
+        This type represents a set of shorts. The string representation
+        is parsed as a set of comma (',') separated shorts that are parsed
+        using the <code>java.lang.Short</code> class.
+      </p>
+      <p>
+        Alternatively, a property of this type can be supplied multiple times which
+        is equivalent to a set of comma separated byte values.
```
Comment (strkkk) [inline src/xdocs/property_types.xml:124]: same here

File: src/xdocs/property_types.xml (hunk 1 above)
Comment (gaurabdg) [reply src/xdocs/property_types.xml:63]: Sorry, my bad, done

File: src/xdocs/property_types.xml (hunk 2 above)
Comment (gaurabdg) [reply src/xdocs/property_types.xml:67]: Done

File: src/xdocs/property_types.xml (hunk 3 above)
Comment (gaurabdg) [reply src/xdocs/property_types.xml:86]: Done

File: src/xdocs/property_types.xml (hunk 4 above)
Comment (gaurabdg) [reply src/xdocs/property_types.xml:105]: Done

File: src/xdocs/property_types.xml (hunk 5 above)
Comment (gaurabdg) [reply src/xdocs/property_types.xml:124]: Done

**Pull request merged**